### DISCUSSION TITLE
how to ignore only ports 80, 443 and icmp

### DISCUSSION THREAD

**Discussion by karthikeayan** [cloud-custodian/cloud-custodian#7342] [General]
i need to create a security group policy that will delete the rule if it is open to world, except for ports 80, 443 and all icmp

Can someone help me with the policy to achieve this?

**Comment by kapilt** [comment]
some hints.. OnlyPorts on the ingress filter, remove-permissions action with `matched`

**Comment by cvemula1** [comment]
type: ingress
        OnlyPorts: [80,443]
        Cidr:
          value: "0.0.0.0/0"
